homeassistant/components/incomfort/binary_sensor.py

```diff
@@ -35,7 +35,27 @@ class IncomfortBinarySensorEntityDescription(BinarySensorEntityDescription):
         translation_key="fault",
         device_class=BinarySensorDeviceClass.PROBLEM,
         value_key="is_failed",
-        extra_state_attributes_fn=lambda status: {"fault_code": status["fault_code"]},
+        extra_state_attributes_fn=lambda status: {
+            "fault_code": status["fault_code"] or "-",
```

Comment:
Why not make the fault_code None instead `-`?

Reply:
Then the attr state becomes `Unknown`. But it is not.